test/ContractTestContext.ts

```diff
@@ -18,10 +18,11 @@ import type {
     LeverageEngine,
     LeverageAllocator,
     PositionToken,
-    ParameterStore,
+    // ParameterStore,
```

Comment:
For this PR, leaving the changes needed as commented out to be used as reference to what needs to be done when we migrate other classes